docs/sections/user_guide/cli/tools/execute.rst

```diff
@@ -0,0 +1,64 @@
+``execute``
+===========
+
+The ``uw`` mode for executing external drivers.
+
+.. literalinclude:: execute/help.cmd
+   :language: text
+   :emphasize-lines: 1
+.. literalinclude:: execute/help.out
+   :language: text
+
+For the three required arguments:
+
+* ``--module`` specifies the name of the module providing the driver. The name may be an absolute path (e.g. ``/path/to/driver.py``); a path relative to the current directory (e.g. ``driver.py``, ``../driver.py``, ``sub/dir/driver.py``); or a name appropriate to the Python ``import`` statement (e.g. ``driver``, ``my.package.driver``), provided the directory containing the module is on ``PYTHONPATH`` / ``sys.path``.
+* ``--class`` specifies the name of a class in the above module that implements the driver, which should use one of the classes exported by ``uwtools.api.driver`` as its base class.
+* ``--task`` specifies the name of a method in the above class that implements a :iotaa:`task<blob/main/README.md#tasks>`, decorated with :iotaa:`@task<blob/main/README.md#task>`, :iotaa:`@tasks<blob/main/README.md#tasks>`, or :iotaa:`@external<blob/main/README.md#external>`.
```

Comment:
@christinaholtNOAA et al., I added this to try to explain these arguments more fully. Please let me know if you have feedback.

Also, in the course of testing the third module-name case, where the module name is specified in Python-code `import` style, e.g. just `driver`, when the directory containing `driver` is on `PYTHONPATH`, I discovered that the code for automatically locating the associated `.jsonschema` file broke because we didn't know the path of the module file to use for finding its sibling schema file. So I made some updates in `uwtools.api.driver` so that that now works correctly.